WebRoot/js/zimbraMail/share/model/ZmOrganizer.js

```diff
@@ -1038,6 +1038,16 @@ function() {
 	return color ? [icon,color].join(",color=") : icon;
 };
 
+/**
+ * Gets the icon alter text.
+ * 
+ * @return	{String}	the icon alter text
+ */
+
+ZmOrganizer.prototype.getIconAlterText = function() {
+	return this.name + " " + ZmMsg.icon;
```

Comment:
The word "icon" will be redundant if SR will read out "An image of calendar icon". In that case let's just have "Calendar" as alt text?

Reply:
I checked with NVDA on windows machine. SR is reading out as "Calendar icon", "Inbox icon".
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/23252779/194261006-5bccba43-964f-4dbf-9781-fd5c92208e0f.png">